server/odc-service/src/main/java/com/oceanbase/odc/service/onlineschemachange/pipeline/ProjectStepResultChecker.java

```diff
@@ -181,7 +191,14 @@ private boolean checkStepFailed() {
                 if (currentProjectStepMap.get(stepName).getExtraInfo() != null) {
                     checkerResult.setErrorMsg(currentProjectStepMap.get(stepName).getExtraInfo().getErrorMsg());
                 }
-                isProjectFailed = true;
+
+                int failedTimes = checkFailedTimes.compute(stepName, (k, v) -> v == null ? 1 : ++v);
```

Comment:
what is the check interval? 
if the check interval set as a small value, then we may need change the threshold, how to maintains threshold setting?l

Reply:
Current only encounter  `FULL_TRANSFER` may cause oms project be blocked during data transfer, and the failed result cannot be recovered in general,  the threshold times is enough.

Reply:
use 120 s to record `FULL_TRANSFER` failed